pkg/sdk/go/sdk.go

```diff
@@ -347,6 +301,38 @@ func NewSDK(conf Config) SDK {
 	}
 }
 
+func (sdk mfSDK) sendRequestAndGetBodyOrError(method, url string, data []byte, token, contentType string, expectedResponseCode ...int) ([]byte, errors.SDKError) {
```

Comment:
Why doesn't this replace `sendRequest`?

Reply:
There are 6 `create` functions which use response header "Location" to create ID, instead of using response body.
We need to use another function for this, something like sendRequestAndGetHeadersorError.

Reply:
Please just rename, maybe something like `processRequest`.